packages/react-components/src/components/MessageThread.test.tsx

```diff
@@ -216,11 +216,12 @@ describe('Message should display image and attachment correctly', () => {
         }
       ]
     };
-    const onFetchAttachment = async (attachment: FileMetadata): Promise<AttachmentDownloadResult[]> => {
+    const onFetchAttachment = async (attachments: FileMetadata[]): Promise<AttachmentDownloadResult[]> => {

```

Comment:
Missing message prop?

Reply:
No this one does not require the message prop